pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "cdip_connector"
-version = "0.9.9"
+version = "1.0.0"
```

Comment:
This is fine. Don't feel like you're limited to single digit though (ex. 0.9.10).